src/config_campaigns.h

```diff
@@ -154,6 +154,7 @@ extern struct CampaignsList mappacks_list;
 extern const struct NamedCommand cmpgn_map_commands[];
 extern const struct NamedCommand cmpgn_map_cmnds_options[];
 extern const struct NamedCommand cmpgn_map_cmnds_kind[];
+extern const struct NamedCommand cmpgn_human_player_options[];
```

Comment:
What does this do? Has it anything to do with the console commands?

Reply:
Yes; it's used to identify players in command parameters.